config/routes.rb

```diff
@@ -394,10 +392,18 @@
       end
 
       scope '/references' do
-        get '/start' => 'decoupled_references#start', as: :decoupled_references_start
+        get '/start' => 'decoupled_references/base#start', as: :decoupled_references_start
+
+        get '/type' => 'decoupled_references/type#new', as: :decoupled_references_new_type
+        post '/update-type' => 'decoupled_references/type#create', as: :decoupled_references_create_type
```

Comment:
This is something we can circle back to if we're low on time, but some suggestions about the routing:

```
      get '/type' => 'decoupled_references/type#new', as: :decoupled_references_type
      post '/type' => 'decoupled_references/type#create'
```
Means you only need the one path helper and can use its path for both GETs and POSTs.

Reply:
Same as below 👍